styles/AdvancedEditor.css

```diff
@@ -173,11 +173,19 @@
   text-align: left;
 }
 
+.ProseMirror .box-element .block-section_title h1.empty-node[data-placeholder]::before {
+  font-style: italic !important;
+  font-weight: 400 !important;
+  line-height: 24px; 
+  letter-spacing: -0.369px;
+  font-size: 13.5pt ;
+}
+.ProseMirror .box-element .block-section:first-of-type > section:first-of-type > .block-section_title:first-of-type > h1.empty-node[data-placeholder]::before {
```

Comment:
hmm we should use that selector for the above style!

Reply:
I need to assign different font sizes for both, so this selector is necessary for the default section.